### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: manifest.json [hunk 1]
```diff
@@ -26,14 +32,17 @@
             {
                 "name": "domain",
                 "type": "domain",
+                "example": "/example",
+                "default": "/example",
```
Comment (alexAubin) [inline manifest.json:36]: Uuuuh these are totally not domains and will be ignored anyway

```suggestion
```

File: manifest.json [hunk 2]
```diff
@@ -26,14 +32,17 @@
...
                 "help": {
                     "en": "ZAP must run in the root of this domain. It means no other app can be accessed/run from this domain. We advise to use a dedicated subdomain such as zap.domain.tld",
                     "fr": "ZAP doit être installé à la racine du domaine. Cela implique qu'aucune autre app ne pourra être installée ou accessible sur ce domain. Nous conseillons un sous-domaine dédié par exemple zap.domain.tld."
                 }
             },
             {
                 "name": "admin",
-                "type": "user"
+                "type": "user",
+                "example": "johndoe"
```
Comment (alexAubin) [inline manifest.json:45]:
```suggestion
                "type": "user"
```

Example values for user are not relevant either

**Comment by alexAubin** [comment]
(Otherwise LGTM, though one way would be nice to take care of the chmod 777 the linter is reporting)

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**